src/services/dashboards/widgets/aws-personal-health-dashboard/AwsPersonalHealthDashboardWidget.vue

```diff
@@ -1,17 +1,193 @@
+        { name: 'scheduled_changes', label: i18n.t('Scheduled changes') }, // song-lang
+        { name: 'other_notifications', label: i18n.t('Other notifications') }, // song-lang
+    ],
+    activeTab: 'open_issues',
+    tableFields: computed<Field[]>(() => [ // song-lang
+        { label: 'Event', name: 'event' },
+        { label: 'Region', name: 'region_code' },
+        { label: 'Start time', name: 'start_time' },
+        { label: 'Last update time', name: 'last_update_time' },
+        {
+            label: 'Affected projects',
+            name: 'affected_projects',
+            detailOptions: {
+                enabled: true,
+                type: 'popover',
+            },
+        },
+    ]),
+    tableItems: computed<Data[]>(() => state.data?.map((d) => ({
+        ...d,
+        region_code: storeState.regionItems[d.region_code]?.name,
+    })) || []),
+});
+
+const widgetFrameProps:ComputedRef = useWidgetFrameProps(props, state);
+
+const fetchData = async ():Promise<Data[]> => new Promise((resolve) => {
+    setTimeout(() => {
+        resolve([
+            {
+                event: 'Quicksight Operational Issue',
+                start_time: '2021-09-01 00:00:00',
+                last_update_time: '2021-09-01 00:00:00',
+                region_code: 'us-east-1',
+                affected_projects: ['project-aaa', 'project-bbb'],
+            },
+            {
+                event: 'Quicksight Operational Issue2',
+                start_time: '2021-09-01 00:00:00',
+                last_update_time: '2021-09-01 00:00:00',
+                region_code: 'us-east-1',
+                affected_projects: ['project-aaa', 'project-bbb'],
+            },
+            {
+                event: 'Quicksight Operational Issue3',
+                start_time: '2021-09-01 00:00:00',
+                last_update_time: '2021-09-01 00:00:00',
+                region_code: 'us-east-1',
+                affected_projects: ['project-aaa', 'project-bbb'],
+            },
+        ]);
+    }, 2000);
+});
+
+const initWidget = async (data?: Data[]) => {
+    state.loading = true;
+    state.data = data ?? await fetchData();
+    state.loading = false;
+    return state.data;
+};
+
+const refreshWidget = async () => {
+    state.loading = true;
+    state.data = await fetchData();
+    state.loading = false;
+    return state.data;
+};
+
+const handleRefresh = () => {
+    refreshWidget();
+};
+
+(() => {
+    store.dispatch('reference/region/load');
+})();
+
+defineExpose<WidgetExpose<Data[]>>({
+    initWidget,
+    refreshWidget,
+});
 
 </script>
+<style lang="postcss" scoped>
+.widget-contents {
+    padding: 0 1.5625rem;
+    height: 100%;
+
+    .tab-container {
+        margin-bottom: 0.25rem;
+    }
+
+    .popover-container {
+        padding-top: 0.59375rem;
+        padding-bottom: 0.34375rem;
+        .popover-title {
+            @apply text-gray-500;
+            font-size: 1rem;
+            font-weight: 700;
+        }
```

Comment:
```
.popover-title {
          @apply text-label-lg text-gray-500;
          font-weight: 700;
      }
```
How about writing according to the typography change?

Information can be found on the right side of Figma. Please refer to the picture below.
<img width="492" alt="image" src="https://user-images.githubusercontent.com/19162140/211230006-53372c4b-807b-4f9a-a7fe-1e27d396b16c.png">